### DISCUSSION TITLE
model-viewer in VueJS

### DISCUSSION THREAD

**Discussion by nadavkomo** [google/model-viewer#2122] [Q&A]
amazing product! 
There is any option for using this model-viewer in VuwJS? thanx:)

**Comment by elalish** [comment]
I believe people have; please search the discussions for help. I'm not an expert on that framework.

**Comment by agriffi10** [comment]
Yes you can! You just have to make sure you add it as an ignored element to your Vue config (both in Nuxt and regular Vue, you'll also have to disable SSR in Nuxt for it).
